express-middlewares/src/index.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const { languages } = require('./languages');
+const { packages } = require('./packages');
```

Comment:
Won't it be better if we put 'languages' and "packages" both in a single "constants" file? Or inside a "Constant" folder?